django/db/models/fields/__init__.py

```diff
@@ -665,11 +665,11 @@ def set_attributes_from_name(self, name):
         if self.verbose_name is None and self.name:
             self.verbose_name = self.name.replace('_', ' ')
 
-    def contribute_to_class(self, cls, name, virtual_only=False):
+    def contribute_to_class(self, cls, name, private_only=False):
```

Comment:
Could we add some comments (perhaps at least a docstring here that explains what "private_only" does)? See previous confusion and uncertainty in https://code.djangoproject.com/ticket/24749.